Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -76,8 +77,8 @@ bool AVIDump::Start(int w, int h)
 {
   s_pix_fmt = AV_PIX_FMT_RGBA;
 
-  s_width = w;
-  s_height = h;
+  s_width = g_Config.iDumpWidth ? g_Config.iDumpWidth : w;
```

Comment:
I think cropping is closer to what appears on a real TV screen. If you've got the aspect ratio right (4:3 or 16:9), then the amount cropped _should_ be pretty small.

Letterboxing would be able to handle switches in aspect ratio while dumping. Is there ever a case where a game switches aspect ratio (for a cutscene or menu, maybe) without user intervention?